app/components/account/action/add/account-create-basic-dialog.html

```diff
@@ -0,0 +1,87 @@
+<bl-complex-form [formGroup]="form" [submit]="submit" [containerRef]="sidebarRef">
+    <bl-form-page main-form-page title="Create batch account" [formGroup]="form">
+        <bl-form-section title="General info" subtitle="Basic information about the batch account">
+            <div class="form-element">
+                <mat-form-field>
+                    <input matInput formControlName="name" placeholder="Account name" />
+                    <mat-hint align="end" *ngIf="selectedLocation">{{accountUrlSuffix}}</mat-hint>
+                </mat-form-field>
+                <bl-error controlName="name" code="required">Account name is a required field</bl-error>
+                <bl-error controlName="name" code="minLength">Account name has a minimum length of 3 characters</bl-error>
+                <bl-error controlName="name" code="maxlength">Account name has a maximum length of 64 characters</bl-error>
+                <bl-error controlName="name" code="pattern">Account name can contain lower case letters and numbers</bl-error>
+                <bl-error controlName="name" code="accountExists">{{ form.controls.name.errors?.accountExists?.message }}</bl-error>
+                <bl-error controlName="name" code="serverError">{{ form.controls.name.errors?.serverError }}</bl-error>
+            </div>
+            <div class="form-element">
+                <mat-form-field>
+                    <mat-select formControlName="subscription" placeholder="Subscription">
+                        <mat-option
+                            *ngFor="let subscription of subscriptionService.subscriptions | async;trackBy: trackBySubscriptionId"
+                            [value]="subscription">
+                            {{subscription.displayName}}
+                        </mat-option>
+                    </mat-select>
+                </mat-form-field>
+                <bl-error controlName="subscription" code="required">Subscription is a required field</bl-error>
+            </div>
+            <div class="form-element" *ngIf="selectedSubscription">
+                <mat-radio-group formControlName="resourceGroupMode">
```

Comment:
Same here in a new component. Also I think here we could actually use a mat-autocomplete instead of a select and radio button(A bit like the container picker for NCJ)

If you autocomplete to a non existing one then you create it.